common/version.go

```diff
@@ -17,4 +17,4 @@
 package common
 
 // Version of the SDK
-const Version = "0.1.0"
+const Version = "0.2.0"
```

Comment:
Revert this change as it will be done automatically by semantic-release when the PR is merged.